psi4/driver/p4util/python_helpers.py

```diff
@@ -195,8 +195,8 @@ def _core_wavefunction_from_file(wfn_data):
         wfn_dimension[label] = core.Dimension.from_list(tup, name=label) if tup is not None else None
 
     for label in wfn_matrixarr:
-        array = wfn_dimension[label]
-        wfn_dimension[label] = core.Matrix.from_array(array, name=label) if array is not None else None
+        array = wfn_matrixarr[label]
```

Comment:
Surprised this passed tests.

Reply:
only hits when there's array psivars. they're pretty rare.